integreat_cms/cms/templates/pages/page_pdf.html

```diff
@@ -87,7 +87,13 @@ <h1 id="title_page">{{ language.table_of_contents }}</h1>
                         {% endif %}
                         {% get_public_translation page language.slug as page_translation %}
                         <h1 class="level-{{ page.depth|add:"-1" }}">{{ page_translation.title }}</h1>
-                        <div class="content" style="padding-bottom: 30px;">{{ page_translation.content|pdf_strip_fontstyles|safe }}</div>
+                        <div class="content" style="padding-bottom: 30px;">
+                            {% if page.mirrored_page_first %}{{ page_translation.mirrored_translation_text|pdf_strip_fontstyles|safe }}{% endif %}
+                            {{ page_translation.content|pdf_strip_fontstyles|safe }}
+                            {% if not page.mirrored_page_first %}
+                                {{ page_translation.mirrored_translation_text|pdf_strip_fontstyles|safe }}
```

Comment:
Since you're not using `display_content` I assume you don't want to show the fallback texts in the PDF - which makes sense to me - but the `mirrored_translation_text` also might return a fallback text when the mirrored translation does not exist... so maybe it would be better to use a different property here which is just empty when the mirrored content does not exist? :thinking:

Reply:
I decided to use this tag because I think it could be confusing if some part of a page would just silently not render, and the error message would still be helpful in pdfs to inform the reader.
If you want though, I can change it to not render the message